### PR TITLE
I can't believe it's butter

### PR DESCRIPTION
**Pull request by tenmillimaster**
#### Summary

SUMMARY: Content "Adds a number of dairy products."

#### Purpose of change
Currently the only way to preserve milk is to dehydrate it. This adds a number of dairy derivatives for use in future recipes, which I needed to do because there are **so many of them** and I was trying to put them all into a PR. Hence this one separate pr to establish them and better focus.

#### Describe the solution
Adds a new sealable preserving container, a sealed tetra-pak style carton, in addition to UHT milk, buttermilk, Butter and ghee, all of which have different shelf lives compared to milk. Raw milk has also been added (with cream in it, so it's more caloric).

I have *not* made cows excrete said milk as there needs to be a way to boil it. Recipes to come, in which cows will make raw milk.

#### Describe alternatives you've considered
Food items I also wanted to add:frog milk, which needs more work identifying just how protective those peptides are and if they're in enough species.
And adding all the various churning types, but that requires more research for efficiencies and yields.

#### Additional context
I hope you are all ready for the horror that is fairy bread.

### CODE REVIEW

**Comment by KorGgenT** [comment]
it looks like you messed up the body of your description. it needs a proper summary, as well.

**Comment by tenmillimaster** [comment]
@KorGgenT that's because I haven't written it yet and needed to do so whilst mobile, where I can't submit PR's (to my knowledge).

File: data/json/item_groups.json [hunk 1]
```diff
@@ -9835,4 +9850,4 @@
       [ "schematics_searchlight", 50 ]
     ]
   }
-]
+]
```
Comment (ZhilkinSerg) [inline data/json/item_groups.json]:
```suggestion
]

```

File: data/json/itemgroups/food_service.json [hunk 2]
```diff
@@ -1037,4 +1047,4 @@
       [ "hacksaw", 10 ]
     ]
   }
-]
+]
```
Comment (ZhilkinSerg) [inline data/json/itemgroups/food_service.json]:
```suggestion
]

```

File: data/json/items/comestibles/dairy.json [hunk 3]
```diff
@@ -145,5 +249,28 @@
...
+    "material": [ "milk", "junk" ],
+    "primary_material": "milk",
+    "volume": 1,
+    "phase": "liquid",
+    "charges": 10,
+    "vitamins": [ [ "vitA", 2 ], [ "vitC", 1 ], [ "calcium", 7 ] ],
+    "fun": 3
   }
-]
+]
```
Comment (ZhilkinSerg) [inline data/json/items/comestibles/dairy.json]:
```suggestion
]

```

File: data/json/items/containers.json [hunk 4]
```diff
@@ -1064,4 +1096,4 @@
     "seals": true,
     "properties": [ [ "burst_when_filled", "75" ] ]
   }
-]
+]
```
Comment (ZhilkinSerg) [inline data/json/items/containers.json]:
```suggestion
]

```

**Comment by ZhilkinSerg** [comment]
Git needs an empty line at the end of the each file.

**Comment by tenmillimaster** [comment]
> Git needs an empty line at the end of the each file.
Got it. Thanks!

**Comment by adamkad1** [comment]
> Git needs an empty line at the end of the each file.

y tho

**Comment by ZhilkinSerg** [comment]
> 
> 
> > Git needs an empty line at the end of the each file.
> 
> y tho

https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

**Comment by Rivet-the-Zombie** [comment]
Now we need some recipes to make these tasty new foods.

**Pull request merged**